test/example-types/metricGroups/pivot.test.js

```diff
@@ -12,33 +10,76 @@ let columnResponse = require('./pivotData/columnResponse')
 let columnResult = require('./pivotData/columnResult')
 let stringify = require('json-stable-stringify')
 
+let pivotTestNode = node => ({
+  key: 'test',
+  type: 'pivot',
+  values: [{ type: 'sum', field: 'LineItem.TotalPrice' }],
+  rows: [
+    { type: 'fieldValues', field: 'Organization.State' },
+    { type: 'fieldValues', field: 'Organization.NameState' },
+  ],
+  columns: [{ type: 'dateInterval', field: 'PO.IssuedDate', interval: 'year' }],
+  extensions: [],
+  ...node,
+})
+
+let getStats = () => {}
+
+let rowsExpansion = {
+  type: 'rows',
+  drilldown: ['New York'],
+  loaded: false,
+}
+
+let rootPivotRequest = {
+  type: 'rows',
+  columns: {
+    drilldown: [],
+    totals: true,
+  },
+  rows: {
+    drilldown: [],
+    totals: true,
+  },
+}
+
 // pass aggsForValues in each stage
 describe('pivot', () => {
   it('aggsForValues', () => {
+    let { getAggsForValues } = createPivotScope(
+      pivotTestNode(),
+      testSchema('LineItem.TotalPrice'),
+      getStats
+    )
     let values = [
       { type: 'min', field: 'LineItem.TotalPrice' },
       { type: 'max', field: 'LineItem.TotalPrice' },
       { type: 'avg', field: 'LineItem.TotalPrice' },
       { type: 'sum', field: 'LineItem.TotalPrice' },
     ]
-    expect(aggsForValues(values)).toEqual({
+    expect(getAggsForValues(values)).toEqual({
       'pivotMetric-min-LineItem.TotalPrice': {
-        min: { field: 'LineItem.TotalPrice' },
+        min: { field: 'LineItem.TotalPrice.untouched' },
```

Comment:
This isn't right - hopefully it's just a bug in the test code, but we do not expect `.untouched` here.

Reply:
Fixed